src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "./Footer.css";
+
+function Footer() {
+  return (
+    <footer className="center-div">
+      <div className="footer-content p-4">
+        <p className="pt-2">
+          Made with{" "}
+          <span className="footer-icon">
+            <i className="fas fa-heart"></i>
+          </span>{" "}
+          by Sanket Dhabarde
+        </p>
+        <div className="footer-link mt-2 center-div">
+          <a
+            className="m-1"
+            href="https://github.com/SanketDhabarde"
+            target="_blank"
+          >
+            <i className="fab fa-github"></i>
+          </a>
+          <a
+            className="m-1"
+            href="https://twitter.com/SanketDhabarde1"
+            target="_blank"
+          >
+            <i className="fab fa-twitter"></i>
+          </a>
+          <a
+            className="m-1"
+            href="https://www.instagram.com/sanket_dhabarde/"
+            target="_blank"
+          >
+            <i className="fab fa-instagram"></i>
+          </a>
+          <a
+            className="m-1"
```

Comment:
You can create a common component for it and pass changing values as props, or you can create a list of object and iterate it, to minimise no. of lines of code.

Reply:
Actually. I am not going to reuse this. So thought of keeping it this way without creating components. But it makes sense to create components. I will do it thanks for the suggestion.